.github/workflows/gradle-wrapper-validation.yml

```diff
@@ -14,4 +14,4 @@ jobs:
     steps:
       - uses: actions/checkout@v4.1.2
 
-      - uses: gradle/wrapper-validation-action@v3.3.0
\ No newline at end of file
+      - uses: gradle/actions/wrapper-validation@v3
```

Comment:
I think gdi spec requires using `v3.3.0` instead of `v3`

Reply:
ok I'll update